packages/tooltip/src/vaadin-tooltip-mixin.js

```diff
@@ -694,6 +726,14 @@ export const TooltipMixin = (superClass) =>
       }
     }
 
+    /** @private **/
+    __importMarkdownHelpers() {
```

Comment:
nit: could be a private static method as well, it would allow us to call it only once in tests e.g. like this:

```js
before(async () => {
  await Tooltip._importMarkdownHelpers();
})
```